src/integrationTest/java/org/opensearch/security/http/OnBehalfOfJwtAuthenticationTest.java

```diff
@@ -141,6 +161,25 @@ public void shouldNotAuthenticateForNonAdminUserWithoutOBOPermission() {
         }
     }
 
+    @Test
+    public void shouldNotIncludeHostMappingInOBOToken() {
```

Comment:
nit: rename test to `shouldNotIncludeRolesFromHostMappingInOBOToken`

Reply:
Fixed